roles/beaker/tasks/main.yml

```
@@ -0,0 +1,46 @@
+---
+- name: Install podman-docker
+  package:
+    name: podman-docker
+    state: installed
+
+- name: Start podman
+  service:
+    name: podman
+    state: started
+
+- name: Install git
+  package:
+    name: git
+    state: installed
+
+- name: Enable ruby 2.7 module
+  command: dnf module enable -y ruby:2.7
+  when: ansible_distribution_major_version == "8"
+
+- name: enable powertools
+  command: dnf config-manager --set-enabled powertools
+  when: ansible_distribution_major_version == "8"
+
+- name: Install Ruby
+  package:
+    state: installed
+    name:
+      - ruby
+      - ruby-devel
+      - rubygem-bundler
+      - gcc-c++
+      - make
+      - redhat-rpm-config
+      - libyaml-devel
+
+- name: Clone puppet module
+  ansible.builtin.git:
+    repo: "https://github.com/theforeman/{{ beaker_puppet_module }}.git"
+    dest: "/src/{{ beaker_puppet_module }}"
+    version: "{{ beaker_puppet_module_version | default('master') }}"
```

```suggestion
    version: "{{ beaker_puppet_module_version | default(omit) }}"
```

will that take the default branch, whichever it is?